pkg/controller/cache/managed_test.go

```diff
@@ -382,6 +401,8 @@ func TestObserve(t *testing.T) {
 
 	for name, tc := range cases {
 		t.Run(name, func(t *testing.T) {
+			a := name
+			fmt.Println(a)
```

Comment:
@muvaf I suspect this was left in by mistake. Can you remove it before I merge?

Reply:
@negz good catch! just removed.